src/terminal/terminal/display.h

```diff
@@ -44,6 +44,17 @@
  */
 #define GUAC_TERMINAL_MAX_CHAR_WIDTH 2
 
+/**
+ * The size of margins between the console text and the border in mm.
+ */
+#define GUAC_TERMINAL_MARGINS 2
```

Comment:
Why not just have a single constant for the margin in inches? I love the metric system as much as anyone else, but it seems a bit overly complicated to use 2 constants to convert mm to inches every time, rather than just working with inches directly.

Reply:
Note: this isn't a demand for a change; I'm just curious. If you really prefer it this way, I'm ok with keeping it.

Reply:
I figured if we wanted to make it adjustable we'd remove the `GUAC_TERMINAL_MARGINS` and add it to the options are something, and it's much easier to think of adjusting in terms of mm rather than 0.0393701 inches

Reply:
Makes sense to me.